cloud_function/main.py

```diff
@@ -117,7 +117,8 @@ def submit_aou_workload(event, context):
     chip_well_barcode = notification.get('chip_well_barcode')
     analysis_version = notification.get('analysis_version_number')
     print(f'Upload complete for {chip_well_barcode}-{analysis_version}')
-    workload_uuid = get_or_create_workload(headers)
+    environment = notification.get('environment')
```

Comment:
```suggestion
    environment = notification.get('environment').lower()
```

I've personally seen a few messages capitalize the first letter of the environment. I don't know what of our stuff is case sensitive but I don't want to find out lol

Reply:
Lol that is a very good point